### PR TITLE
[wasm] rename WasmApp.* to BrowserWasmApp.*

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

**Comment by pavelsavara** [comment]
next step rename also `src/mono/wasm/` -> `src/mono/browser/` ? 
Or keep it for common stuff and move browser files.

**Comment by pavelsavara** [comment]
And maybe also `src\mono\wasm\runtime\` to `src\mono\browser\javascript\` ?

File: src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj [hunk 1]
```diff
@@ -15,8 +15,8 @@
     <PackageFile Include="$(RepoRoot)\src\mono\wasm\build\ILLink.Substitutions.*.xml" TargetPath="Sdk" />
     <PackageFile Include="$(RepoRoot)\src\mono\wasm\build\ILLink.Descriptors.*.xml" TargetPath="Sdk" />
 
-    <PackageFile Include="$(RepoRoot)\src\mono\wasm\build\WasmApp.props" TargetPath="Sdk" />
-    <PackageFile Include="$(RepoRoot)\src\mono\wasm\build\WasmApp.targets" TargetPath="Sdk" />
+    <PackageFile Include="$(RepoRoot)\src\mono\wasm\build\BrowserWasmApp.props" TargetPath="Sdk" />
```
Comment (pavelsavara) [inline src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj:18]: Naive question, are any of the msbuild files referred to by filename in the SDK ?

**Review by pavelsavara — APPROVED** [review]

**Review by maraf — APPROVED** [review]

File: src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj (hunk 1 above)
Comment (radical) [reply src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj:18]: Good question. A quick search didn't show anything like that. We'll have to react to it, if it breaks anything.

**Pull request merged**